Makefile

```
@@ -737,16 +737,21 @@ init-submodules-e: init-webapps-submodules-e
 
 .PHONY: update-vendor
 update-vendor:
-	# update modules in api/
-	cd api && go mod tidy
 	# update modules in root directory
 	go mod tidy
 	go mod vendor
-	# delete the vendored api package. In its place
-	# create a symlink to the the original api package
+	make update-vendor-api
+
+# update-vendor-api runs go mod tidy and creates a symlink between 
+# /vendor/github.com/gravitational/teleport/api/v7 and /api.
+# This is used to avoid vendoring the local api submodule.
+.PHONY: update-vendor-api
+update-vendor-api:
+	# update modules in api/
+	cd api && go mod tidy
 	rm -r vendor/github.com/gravitational/teleport/api/v7
```

If the directory is not absolutely guaranteed to exist the following would be safer:
```suggestion
	rm -rf vendor/github.com/gravitational/teleport/api/v7
```